pkgs/tools/security/ripasso/cursive.nix

```
@@ -1,26 +1,29 @@
-{ stdenv, lib, rustPlatform, fetchFromGitHub, pkg-config, ncurses, python3, openssl, libgpg-error, gpgme, xorg, AppKit, Security, installShellFiles }:
+{ stdenv, lib, rustPlatform, fetchFromGitHub, pkg-config, python3, openssl, libgpg-error, gpgme, xorg, nettle, llvmPackages, clang, AppKit, Security, installShellFiles }:
```

```suggestion
{ stdenv, lib, rustPlatform, fetchFromGitHub, pkg-config, python3, openssl, libgpg-error, gpgme, xorg, nettle, clang, AppKit, Security, installShellFiles }:
```
llvmPackages is prob not needed

---

Woops, missed out on the reference to llvmPackages in LIBCLANG_PATH. My bad.